Tools/Taxonomy/Common/Constants.cs

```diff
@@ -12,6 +14,10 @@ private static class Guid
             public static string CWE_7PK_V4_5 = "B3D28502-8E8C-43E2-AE32-F8E79C7245D6";
             public static string CWE_Comprehensive_V43 = "FFC64C90-42B6-44CE-8BEB-F6B7DAE649E5";
             public static string CWE_Comprehensive_V44 = "7D699257-C37D-4A10-9C98-D7DB481F1A8B";
+            public static string CWE_Comprehensive_V45 = "474AE5EE-3D14-4862-B691-00658507100F";
+            public static string CWE_Comprehensive_V46 = "D9175477-7CDB-4A3B-9037-3D913B16947A";
+            public static string CWE_Comprehensive_V47 = "CC7C15FF-3B88-4B86-A2DB-55A4DD8F6200";
+            public static string CWE_Comprehensive_V48 = "A70FCFF4-F2E9-4210-9D2A-BF50D7D64D5D";
```

Comment:
>V48 [](http://example.com/codeflow?start=51&length=3)

nit: add a separate between main/minor version number. E.g. V4_8. To be consistent with other constants. Also easier to read.

Reply:
thanks, fixed